modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -314,7 +315,9 @@ export default class BaseTssUtils<KeyShare> extends MpcUtils implements ITssUtil
    * Checks whether the third party backup provider is valid/supported
    * @param backupProvider - the backup provider client selected
    */
-  isValidThirdPartyBackupProvider(backupProvider: string | undefined): boolean {
+  isValidThirdPartyBackupProvider(
+    backupProvider: BackupProvider | string | undefined
```

Comment:
should this not have string then

Reply:
Its fine to have a string here, if this function returns true it just says that the input string is a valid BackupProvider.